k8s/externalsnapshotter/externalsnapshotter.go

```diff
@@ -0,0 +1,147 @@
+package externalsnapshotter
+
+import (
+	"fmt"
+	"os"
+	"sync"
+
+	"github.com/kubernetes-csi/external-snapshotter/v2/pkg/client/clientset/versioned/typed/volumesnapshot/v1beta1"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+var (
+	instance Ops
+	once     sync.Once
+)
+
+// Ops is an interface to perform operations on external storage resources.
+type Ops interface {
+	SnapshotOps
```

Comment:
We need to add SnapshotContentOps and SnapshotClassOps here

Reply:
Updated.